### ISSUE TITLE
prepublish scripts are deprecated

### ISSUE THREAD

**Issue by DanielRuf** [digiaonline/react-foundation#59]
```
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts are deprecated.
npm WARN prepublish-on-install Use `prepare` for build steps and `prepublishOnly` for upload-only.
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.
```

**Comment by DanielRuf** [comment]
Open PR: https://github.com/digiaonline/react-foundation/pull/60

**Comment by hugovk** [comment]
Fixed by #60.
